lychee-bin/tests/cli.rs

```diff
@@ -606,6 +606,80 @@ mod cli {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn test_lycheecache_file() -> Result<()> {
+        let mut cmd = main_command();
+
+        let test_base_path = fixtures_path().join("cache");
+        assert!(
+            fs::metadata(test_base_path.join(".lycheecache")).is_err(),
+            ".lycheecache file should not exist before this test"
+        );
```

Comment:
There's also https://doc.rust-lang.org/std/path/struct.Path.html#method.exists

```rust
assert!(
    !test_base_path.join(".lycheecache").exists(),
    ".lycheecache file should not exist before this test"
);
```

Reply:
Under the hood it does exactly the same, but it's arguably a bit easier to read.

Reply:
That's much nicer! I'll update it to use `exists`